scripts/build/windows.bat

```diff
@@ -226,6 +233,10 @@ signtool verify /pa /v %package_output%\Etcher.exe
 
 upx -9 %package_output%\*.dll
 
+cd %package_output%^
+ && 7z a -tzip ..\installers\%package_name%.zip *^
```

Comment:
> I'm not sure I'd count 7-zip as "sketchy" ? :-S

I think a large part of my aversion stems from the fact that they used to host their downloads on SourceForge and redirected you there to download. Even today, they have links pointing there, and their web page is ugly and unkempt to the point where I still wouldn't download it unless forced to.

The more important idea here is that minimizing dependencies is important. Adding a dependency means adding one more thing that a contributor must download and means opening the process up to more uncontrollable failure modes. For example, what happens when an unknowing contributor installs 7-zip (as prompted by the failing build message) but fails to check the "Add to path" option? They'll continue getting that error even though they have installed the app already. This kind of an issue can easily dissuade prospective contributors.

> As a side note, the VBS script doesn't seem to work at all, and the PowerShell one only works on Windows 8 and newer.

Hmmm, that's unfortunate. I'd contend that people shouldn't be on pre-Windows 8 versions anyway, but in general that isn't a maintainable position 😞 In that case, including the binary seems like the best bet.

Honestly, it might make sense for you to take the binaries for as many of Etcher's build deps as possible and push them to a server so that they could then be downloaded during the build, greatly simplifying the work on a contributor. That's a larger-scale plan though.